Makefile

```
@@ -5,9 +5,9 @@ fmtcheck:
 	(! gofmt -s -d . | grep '^')
 
 vet:
-	go vet ./...
+	go vet ./... && cd v2 && go vet ./...
```

`./...` は再帰的にすべてのファイルを検索してくれるので変更する必要はありません。
```suggestion
	go vet ./...
```